pkg/dogstatsd/server.go

```diff
@@ -341,6 +347,39 @@ func NewServer(demultiplexer aggregator.Demultiplexer, serverless bool) (*Server
 		}
 	}
 
+	// Creating a new logger
+	cfg := seelogCfg.NewSeelogConfig(
+		"dogstatsd",
+		"info",
+		"common",
+		"",
+		"%Date %Time | [DOGSTATSD_STATS] | [%LEV] %Msg ",
+		false)
+
+	// Configuring the log file path
+	logFile := config.Datadog.GetString("dogstatsd_log_file")
+	if logFile == "" {
+		logFile = constants.DefaultDogstatsDLogFile
+	}
+
+	// Configuring max roll for log file, if dogstatsd_log_file_max_rolls env var is not set within datadog.yaml then default value is 3
+	dogstatsd_log_file_max_rolls := config.Datadog.GetInt64("dogstatsd_log_file_max_rolls")
+	if dogstatsd_log_file_max_rolls == 0 {
```

Comment:
What if `dogstatsd_log_file_max_rolls` is less than zero?

Would it make sense to emit a log warning if the value is invalid? I think we do so in other places.